### ISSUE TITLE
Andrew's comments on main text

### ISSUE THREAD

**Issue by hannahm8** [daccordeon/gravexplain#24]
Comments from Andrew on main text

- [x] abstract - "In the accompanying Supplementary..." - please reword, it's a bit awkward, e.g. unclear what words are verbs/nouns

- [x] general - now that we moved stuff around, please check that appendices are numbered in the order they are first mentioned in main text

- [x]  sec III, para 1, first sentence - ref JKS98 for monochromatic claim

- [x] sec IV, para 1, 1st sentence - seems to contradict sec III - I think what we really mean is "Although continuous grav wave signals are nearly monochromatic, the long observing times (~ 1 yr) mean that searches are sensitive to very small frequency drifts. A typical observation involves ~1e10 wave cycles, and a coherent search must track the phase to better than half a cycle over the full observation."

- [ ] sec IVA, first para - since ref mentioned other searches, it would be good to have 1-2 brief sentences accompanying the references which mention what those "other searches" are (broad categories, for instance, just so that there is some guidance for the reader)

- [x] sec IVA - emission matrix - I thought we always say "emission probability"

- [x] para after eq (2) - faster than the algorithm is allowed to - I think we mean "faster than our choice of A(f_k,f_m) allows"

- [x] para before sec VA - Instead -> In addition

- [x] sec VC - beyond the unfiltered recording - what does this mean?

- [x] start new para at "We find some improvement with music over speech..."

- [x]  and in future work fix coincidently -> ally

- [x] worth mentioning somewhere that the LIGO control system has dynamic feedback loops which are very complicated and outside the scope of the table-top

- [x] eq (C2) needs brackets cos(wt+phi)

- [x] last line of appendix C - typo - would reword as "plus the term <x,x>, which is independent of \omega and \phi and can therefore be ignored when searching over \omega"

- [x] pseudocode in appendix D - please check it (again)! - I'm nervous about leading any teachers astray via an accidental bug!

**Comment by hannahm8** [comment]
@daccordeon - for this comment above, 
"sec VC - beyond the unfiltered recording - what does this mean?"
Should we instead say something like this?
"in comparison to the unfiltered recording"

**Comment by daccordeon** [comment]
> All three of these cases can, in extreme cases, worsen speech intelligibility beyond the unfiltered recording and so we choose filters that compromise between achieving the three conditions.

I mean that they can be worse than doing nothing at all. I have rewritten it to say:
"All three of these cases can, when taken to the extreme, make the speech intelligibility worse than the unfiltered recording, i.e.\ they can be worse than doing no filtering. Therefore, we choose filters that compromise between achieving the three conditions."

**Comment by daccordeon** [comment]
I've responded to Andrew's comments on the appendices.

**Comment by hannahm8** [comment]
Added some text separating targeted/directional searches and all sky. 

Closing issue
